packages/compass-components/src/components/content-with-fallback.tsx

```diff
@@ -114,12 +114,14 @@ const fallbackContainer = css({
   bottom: 0,
   left: 0,
   opacity: 0,
-  transition: 'opacity .16s ease-out',
+  transition: 'opacity display .16s ease-out',
```

Comment:
i almost feel this can just be confusing at this point, should we just drop it?

Reply:
unless i'm missing something ..

Reply:
this has a nice side-effect that the visibility(coming into view) of fallback content still gets animated nicely.

Reply:
actually there very little difference (just checked again), maybe i will just remove it to cut down any confusion here.

Reply:
done 👍